### ISSUE TITLE
[Word Universal]: NVDA is not announcing the list position as well as list level (nesting).

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8289]
###Steps to reproduce:
1. Open Sample doc
2. Use Down arrow and navigate to some list
3. Observe that NVDA is not announcing list type of most of the bullets. (Square, Dash-)
4. Also observed that there is no announcement of list position.
5. Observe that there is no announcement of list level (Nesting)

### Expected behavior:
NVDA should announce the list position and list level (nesting).

### Actual behavior:
NVDA is not announcing the list position as well as list level (nesting).

### System configuration:
NVDA version: 2018.1.1
App Name: Word
NVDA Installed or portable: Installed
**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by Adriani90** [comment]
@WiproPSAccTest what do you mean with word universal? Is this word 365? If so, reading the bullets in a list should actually work properly if:
1. You set the punctuation to "all" in NVDA's voice Settings 
2. You enable lists in NVDA's "document formating Settings".
For this Problem there are several issues already in place.

**Comment by Adriani90** [comment]
No updates from the initial author and the list level issue is already covered in other issues already. Closing this as duplicate of i.e. #6815
